### PR TITLE
Update BigQuery notebook

### PR DESCRIPTION
**Pull request by alecglassford**
* Fix links
* Update for style
* Use latest URIs for pre-built containers and adjust code slightly to pass test

### CODE REVIEW

File: ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb [hunk 1]
```diff
@@ -29,22 +29,21 @@
         "id": "title"
```
Comment (ivanmkc) [inline ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:29]: <pre class="ql-syntax" spellcheck="false">Line #4.    TRAIN_IMAGE = "us-docker.pkg.dev/vertex-ai/training/{}:latest".format(TRAIN_VERSION)
</pre><p>Why the change?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/394/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

**Comment by ivanmkc** [comment]
The BQ python library is very finicky from my experience. The new Docker image may have changed the google-cloud-storage version so you might have to force it to install a certain version when running the training job. See how I forced the BigQuery client library version.

File: ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb (hunk 1 above)
Comment (alecglassford) [reply ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:29]: <p>This is the new repository for pre-built container images based after the "Vertex AI" branding was introduced. See https://cloud.google.com/vertex-ai/docs/training/pre-built-containers</p><p> </p><p>I expect that the image you get from the old URI and the new one should be identical. (If not, that's a problem.)</p>

Comment (ivanmkc) [inline ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:29]: <p>Shall we fix this links in the notebook while we're at it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/394/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (ivanmkc) [inline ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:29]: <p>Perhaps emphasize that we do it during prediction as well:</p><p> </p><p>Later during prediction, use these summary statistics again to normalize the testing data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/394/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

**Review by ivanmkc — APPROVED** [review]
A few comments but LGTM

File: ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb (hunk 1 above)
Comment (alecglassford) [reply ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:29]: <p>I think the updated links are correct? They're only showing up weird in ReviewNB. They look OK here: https://github.com/GoogleCloudPlatform/ai-platform-samples/blob/update-bq-notebook/ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb</p>

Comment (alecglassford) [reply ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb:29]: <p>Done.</p>

**Pull request merged**